README.md

```diff
@@ -6,36 +6,57 @@ Some tools and scripts for a UPS test suite
 ### WireMock
 In order to stress-testing we must not talk directly to FCM, but to a mocked server that is provided by [WireMock](http://wiremock.org/) tool.
 
-### Mocked Data Loader
+### Mock APNs
+In case of iOS devices we need to use a mock of the APNs, in this case provided by [Mock APNS Server](https://github.com/aerogear/mockapns).
 
+### Mocked Data Loader
 A tool to produce mock data (push applications, variants and tokens) can be found into the [mock-data-loader](mock-data-loader) folder.
 
-
 ### Artillery
 [Artillery](https://artillery.io/) is a command-line load testing tool that will send requests directly to UPS's REST API. 
 
 ## Usage
-First step is to install and startp up the mocked FCM server. In order to do it, `cd` to the path where you want to install it and run:
+First step is to install and start up both mocked servers.
+#### FCM server
+Firstly `cd` to the path where you want to install it and simply run the next script, it will create the server inside a folder and start it:
 ```
 bash <(curl https://gist.githubusercontent.com/josemigallas/9577750d09f87aaaa570c64d5ce8b58e/raw/83124a8596c93a862bcaefbb2dad4522c5d60828/Start%2520Up%2520WireMock)
 ```
-This script will create the server and start it.
-To start it again use:
+
+To start it again only use:
 ```
 java -jar path/to/server/wiremock-standalone-2.4.1.jar --port 3000
 ```
-Once it is running, start your UPS setting `-Dcustom.aerogear.fcm.push.host` parameter with the fake URL:
+#### APNs server
+After this, install and run the APNs server following [this instructions](https://github.com/aerogear/mockapns).
+
+Now, before starting UPS you will need to add the APNs Mock Server certificate to your JVM trusted certificates list. In order to do that convert it to DER format
+```
+openssl x509 -in path/to/certificate -out cert.crt -outform DER
 ```
-path/to/jboss/bin/standalone.sh -b 0.0.0.0 --server-config=standalone-full.xml -Dcustom.aerogear.fcm.push.host=http://localhost:3000/fcm/send
+
+Then go to your JVM home directory and import the ssl certificate into the trusted certificates list:
+```
+cd $JAVA_HOME/jre/lib/security
+sudo keytool -importcert -keystore cacerts -storepass changeit -file /path/to/cert.crt -trustcacerts
+```
+> MacOS: run `$ /usr/libexec/java_home` to locate your $JAVA_HOME dir.
+
```

Comment:
Worth to mention that the user should answer yes to the 'trust the certificate' question